api/MediaStreamTrack.json

```diff
@@ -1041,10 +1041,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
I think we should remove the `readonly` feature entirely. It’s not part of the spec.

Reply:
I think https://chromium.googlesource.com/chromium/src/+/bcc6621834abbd8f95c4d36e18d691ac495b0aa1 explains why this is here. It was never web-exposed in Chrome, but it was an internal concept. It would make sense to remove together with the `remote` entry, I think.